src/js/vre/chapter36/config/form.js

```diff
@@ -85,6 +86,61 @@ const applicantServiceHistory = _.merge(serviceHistory, {
   }
 });
 
+const previousBenefitApplicationsUI = {
+  'ui:title': 'Have you ever applied for any of the following VA benefits? (Check all that apply.)',
+  'ui:options': {
+    showFieldLabel: true
+  },
+  chapter31: {
+    'ui:title': benefitsLabels.chapter31,
+    'ui:options': {
+      hideIf: ({ 'view:isVeteran': isVeteran }) => !isVeteran
+    }
+  },
+  ownServiceBenefits: {
+    'ui:title': benefitsLabels.ownServiceBenefits,
+    'ui:options': {
+      hideIf: ({ 'view:isVeteran': isVeteran }) => !isVeteran
+    }
+  },
+  dic: {
+    'ui:title': benefitsLabels.dic,
+    'ui:options': {
+      hideIf: ({ 'view:isVeteran': isVeteran }) => isVeteran
+    }
+  },
+  other: {
+    'ui:title': benefitsLabels.other
+  },
+  otherExplanation: {
+    'ui:title': benefitsLabels.otherExplanation,
+    'ui:options': {
+      expandUnder: 'other'
+    }
+  }
+};
+
+const previousBenefitApplications = {
```

Comment:
Is the plan to update the schema with this and then pull it from there?

Reply:
Yes, thanks for the reminder. Will update.